tardis/commands/lastlog.py

```diff
@@ -0,0 +1,57 @@
+import tardis.ldaptools
+from collections import defaultdict
+from subprocess import Popen, PIPE
+import re
+
+# lastlog - audit users
+# exit codes:
+#   0   - Normal
+#   100 - audit failed somehow
+
+
+def setup(parser):
+    parser.add_argument("-type", -"t" help="Audit user logins -- abandon for abandonment -- password for password type.")
```

Comment:
This looks it might be a syntax error—should it be `parser.add_argument("-type", "-t", help=...)`? Probably best to use `--type` for the long argument, as well

Reply:
Fair point, I was editing locally and some minor issues weren't changed.

Should now be fixed in b317672